spec/support/magi_medicaid/mitc/eligibility_request.json

```diff
@@ -1,134 +1,130 @@
-           "Pensions and Annuities Taxable Amount":60,
-           "Farm Income or Loss":60,
-           "Unemployment Compensation":60,
-           "Other Income":60,
-           "MAGI Deductions":60
-        },
-        "Relationships":[
-           {
-              "Other ID":2,
-              "Relationship Code":"03"
-           }
-        ]
-     },
-     {
-        "Is Applicant":"Y",
-        "Applicant Attest Blind or Disabled":"N",
-        "Student Indicator":"N",
-        "Medicare Entitlement Indicator":"N",
-        "Incarceration Status":"N",
-        "Lives In State":"Y",
-        "Claimed as Dependent by Person Not on Application":"N",
-        "Applicant Attest Long Term Care":"N",
-        "Has Insurance":"N",
-        "State Health Benefits Through Public Employee":"N",
-        "Prior Insurance":"N",
-        "Applicant Pregnant Indicator":"N",
-        "Applicant Post Partum Period Indicator":"N",
-        "Former Foster Care":"N",
-        "Required to File Taxes":"N",
-        "US Citizen Indicator":"Y",
-        "Person ID":2,
-        "Applicant Age":5,
-        "Hours Worked Per Week":0,
-        "Income":{
-           "Monthly Income":0,
-           "Wages, Salaries, Tips":0,
-           "Taxable Interest":0,
-           "Tax-Exempt Interest":0,
-           "Taxable Refunds, Credits, or Offsets of State and Local Income Taxes":0,
-           "Alimony":0,
-           "Capital Gain or Loss":0,
-           "Pensions and Annuities Taxable Amount":0,
-           "Farm Income or Loss":0,
-           "Unemployment Compensation":0,
-           "Other Income":0,
-           "MAGI Deductions":0
-        },
-        "Relationships":[
-           {
-              "Other ID":1,
-              "Relationship Code":"04"
-           }
-        ]
-     }
+      "Income": {
+        "Monthly Income": 0,
+        "Wages, Salaries, Tips": 25128,
+        "Taxable Interest": 60,
+        "Tax-Exempt Interest": 60,
+        "Taxable Refunds, Credits, or Offsets of State and Local Income Taxes": 60,
+        "Alimony": 60,
+        "Capital Gain or Loss": 60,
+        "Pensions and Annuities Taxable Amount": 60,
+        "Farm Income or Loss": 60,
+        "Unemployment Compensation": 60,
+        "Other Income": 60,
+        "MAGI Deductions": 60
+      },
+      "Relationships": [
+        {
+          "Other ID": 2,
+          "Relationship Code": "03"
+        }
+      ]
+    },
+    {
+      "Is Applicant": "Y",
+      "Applicant Attest Blind or Disabled": "N",
+      "Student Indicator": "N",
+      "Medicare Entitlement Indicator": "N",
+      "Incarceration Status": "N",
+      "Lives In State": "Y",
+      "Claimed as Dependent by Person Not on Application": "N",
+      "Applicant Attest Long Term Care": "N",
+      "Has Insurance": "N",
+      "State Health Benefits Through Public Employee": "N",
+      "Prior Insurance": "N",
+      "Applicant Pregnant Indicator": "N",
+      "Applicant Postpartum Period Indicator": "N",
```

Comment:
_MAGI In The Cloud_ expects us to send the attribute name as 'Applicant Post Partum Period Indicator'. Please revert this change.